### PR TITLE
Update base-de.yaml

### PR DESCRIPTION
**Pull request by Oblivo**

### CODE REVIEW

File: translations/base-de.yaml [hunk 1]
```diff
@@ -44,36 +44,36 @@ steamPage:
...
-            [*] Unlimited Waypoints
-            [*] Unlimited Savegames
-            [*] Additional settings
-            [*] Coming soon: Wires & Energy! Aiming for (roughly) end of July 2020.
-            [*] Coming soon: More Levels
-            [*] Allows me to further develop shapez.io ❤️
+            [*] Unbegrenzt viele Wegpunkte
+            [*] Unbegrenzt viele Savegames
+            [*] Zusätzliche Einstellungen
+            [*] Coming soon: Kabel & Energie! Aiming for (roughly) end of July 2020. Geplant für (ungefähr) ende Julie 2020.
```
Comment (tobspr) [inline translations/base-de.yaml:54]: * Juli

Und die ursprüngliche Übersetzung ist noch da

**Comment by tobspr** [comment]
Approved it by accident, it should have meant "rejected"

**Comment by tobspr** [comment]
There are conflicts, could you have a look?

**Comment by Oblivo** [comment]
> There are conflicts, could you have a look?

I've changed the part whit the spelling mistake and the old english part which I forgot to delete. Was there anything else? I'm not too familiar with github as of right now

**Comment by tobspr** [comment]
TY!

**Pull request merged**